pom.xml

```diff
@@ -160,4 +188,11 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+	<repositories>
```

Comment:
Yes, this repo should already be linked to from the Spring IO repository as "gemstone-release-cache"